src/tools/idlc/src/idlc.c

```diff
@@ -358,6 +372,9 @@ static idl_retcode_t idlc_parse(void)
     ret = idl_parse(pstate);
     assert(ret != IDL_RETCODE_NEED_REFILL);
     if (ret == IDL_RETCODE_OK) {
+      if ((ret = set_type_extensibility()) != IDL_RETCODE_OK)
```

Comment:
Note to self: move setting default extensibility to `idl_parse`, because this is also required for validations when parsing the idl (e.g. must-understand usage wrt extensibility)